pavement.py

```diff
@@ -89,7 +89,7 @@
         GIT_REVISION = "Unknown"
 
     if not setup_py.ISRELEASED:
-        FULLVERSION += '.dev-' + GIT_REVISION[:7]
+        FULLVERSION += '.dev0+' + GIT_REVISION[:7]
```

Comment:
Looks like it won't cause a problem
```
In [7]: NumpyVersion('1.9.0.dev-e24486e') == NumpyVersion('1.9.0.dev0+e24486e')
Out[7]: True

In [8]: NumpyVersion('1.9.0.dev-e24486e') == NumpyVersion('1.9.0.dev0')
Out[8]: True
```
However, this in Scipy probably doesn't do what is expected.
```
_rfft_mt_safe = (NumpyVersion(np.__version__) >= '1.9.0.dev-e24486e')
```